src/game/containers/SocialSharingContainer.js

```diff
@@ -0,0 +1,114 @@
+import { h } from 'preact';
+import { connect } from 'preact-redux';
+import { createStructuredSelector } from 'reselect';
+
+import { hasScore, totalTime, playerPosition, playersCount } from '../selectors/player-score-selectors';
+
+import { puzzlesCount } from '../../shared/selectors/puzzle-selectors';
+import { AuthButton } from '../../components/AuthButton/AuthButton';
+
+function buildTweetText(totalTime, playerPosition, playersCount, puzzlesCount) {
+    let message = `Today I've spent ${totalTime} seconds to solve ${puzzlesCount} puzzles in #CSSQD by @2123io`;
+    if (playerPosition <= 10) {
```

Comment:
And... `else`?

Reply:
I'd rather not tweet anything related to my position in score if it's not so good

Reply:
But we still display the button, right? Once user presses it - we'll tweet `undefined`, I believe

Reply:
fixed